### ISSUE TITLE
About frameworks plan

### ISSUE THREAD

**Issue by ajoeajoe** [openml/automlbenchmark#37]
frameworks worth mentioning as you plan to add:

oboe       added
autoxgboost  no
hyperopt-sklearn  added
ML-Plan  no

I just want to ask when autoxgboost are added as you plan.

and give some info about “ModuleNotFoundError: No module named '_regression’ ”

**Comment by ajoeajoe** [comment]
run python runbenchmark.py autoweka test and errors found：

AutoWEKA failed producing any prediction.
Traceback (most recent call last):
  File "/Users/joe/Documents/AutoML/experiment/automlbenchmark/automl/benchmark.py", line 371, in run
    meta_result = framework.run(self._dataset, task_config)
  File "/Users/joe/Documents/AutoML/experiment/automlbenchmark/frameworks/AutoWEKA/__init__.py", line 10, in run
    return run(*args, **kwargs)
  File "/Users/joe/Documents/AutoML/experiment/automlbenchmark/frameworks/AutoWEKA/exec.py", line 72, in run
    raise NoResultError("AutoWEKA failed producing any prediction.")
automl.results.NoResultError: AutoWEKA failed producing any prediction.
Metric scores: { 'acc': nan,
  'auc': nan,
  'duration': nan,
  'fold': 1,
  'framework': 'AutoWEKA',
  'id': 'openml.org/t/3913',
  'info': 'NoResultError: AutoWEKA failed producing any prediction.',
  'mode': 'local',
  'models': nan,
  'params': '',
  'result': nan,
  'seed': 2189735279,
  'tag': 'stable',
  'task': 'kc2',
  'utc': '2019-07-11T15:37:59',
  'version': '2.6'}

**Comment by ajoeajoe** [comment]
ok it is an problem of pyrfr , refer to https://github.com/automl/auto-sklearn/issues/308

**Comment by seb-h2o** [comment]
@ajoeajoe please better create a new ticket rather than editing the old one. Otherwise we lose the information about your original ticket and especially the solution (that the `ModuleNotFoundError: No module named '_regression'` error was due to the `plyr` version provided with `conda`.
The setup instructions are actually written for a basic Python3 setup: it's very possible to get version incompatibilities when using `conda`.
This is an issue we may want to consider.

**Comment by seb-h2o** [comment]
@ajoeajoe now regarding the plans:
- `oboe` is added but we still need to upgrade and rerun the benchmarks against it: we encountered some problems with the previous version, mentioned them to Oboe team and they recently fixed those issues. So, please don't consider the state of `oboe` support as completed. This should be done before end of July though.
- plan is to add `autoxgboost` during summer too (don't have much time for this right now, but probably in a couple of weeks). If you want to speed up things, then we'll be happy to have you contributing: you can take `autosklearn` as a reference for example. As this one open-source, once we have it integrated and stable, we can run the benchmarks we did run for other frameworks on our side and publish the results on the website.
- for `ML-Plan`, maybe were looking for contributors as well.

**Comment by seb-h2o** [comment]
https://github.com/openml/automlbenchmark/issues/41
https://github.com/openml/automlbenchmark/issues/42
https://github.com/openml/automlbenchmark/issues/43

**Comment by ajoeajoe** [comment]
ok thx and look forward to autoxgboost
